eclair-core/src/main/scala/fr/acinq/eclair/payment/LocalPaymentHandler.scala

```diff
@@ -97,5 +98,7 @@ class LocalPaymentHandler(nodeParams: NodeParams)(implicit ec: ExecutionContext
 }
 
 object LocalPaymentHandler {
-  def props(nodeParams: NodeParams) = Props(new LocalPaymentHandler(nodeParams))
+  def props(nodeParams: NodeParams): Props =  {
```

Comment:
The added brackets seem superfluous

Reply:
This line was getting a little long. I thought it wouldn't be a bad idea to wrap it to a new line and add `{` :-)

Reply:
That's a valid reason, but all other files were doing it differently, so I reverted for consistency